packages/contracts/contracts/libs/v0.8.x/minter-libs/SplitFundsLib.sol

```diff
@@ -96,6 +96,44 @@ library SplitFundsLib {
         }
     }
 
+    /**
+     * @notice pays ETH funds to sender (if refund), with all of token price
+     * being sent to the render provider for a token purchased on project
+     * `projectId`.
+     * WARNING: This function uses msg.value and msg.sender to determine
+     * refund amounts, and therefore may not be applicable to all use cases
+     * (e.g. do not use with Dutch Auctions with on-chain settlement).
+     * @dev This function relies on msg.sender and msg.value, so it must be
+     * called directly from the contract that is receiving the payment.
+     * @dev possible DoS during splits is acknowledged, and mitigated by
+     * business practices, including end-to-end testing on mainnet, and
+     * admin-accepted artist payment addresses.
+     * @param projectId Project ID for which funds shall be split.
+     * @param pricePerTokenInWei Current price of token, in Wei.
+     * @param coreContract Address of the GenArt721CoreContract associated
+     * with the project.
+     */
+    function sendAllToRenderProviderETHRefundSender(
```

Comment:
when would this function get called?

Reply:
we use it in the new minters because we allow the purchaser to send >price (consistent with existing fixed price minters). We could definitely update the logic in these minters to force `msg.value == price`, and then not run through the refund logic.
what do you think?

Reply:
makes sense, I think fine to leave as is! 🙏